3주차/[Lv.3]_야근_지수/이현준.js

```diff
@@ -0,0 +1,14 @@
+function solution(n, works) {
+  if (works.reduce((acc, work) => (acc += work), 0) < n) return 0;
+  works.sort((a, b) => b - a);
+
+  for (let i = 0; i < n; i++) {
+    let j = 0;
+    while (works[j] <= works[j + 1]) {
+      j++;
+    }
+    works[j] -= 1;
+  }
+
+  return works.reduce((acc, work) => (acc += work ** 2), 0);
```

Comment:
```suggestion
  return works.reduce((acc, work) => (acc + work ** 2), 0);
```
계산된 값이 다음 `acc`로 반환되는것으로 `acc + work ** 2`이 맞는 표현인 것 같습니다!! 🤔
```javascript
  return works.reduce((acc, work) => (acc + Math.pow(work, 2)), 0);
```
위와 같이 표현할 수 있어 적어보았습니다!!